CANTroller2/src/globals.h

```diff
@@ -199,27 +195,12 @@ Timer cruiseAntiglitchTimer(cruise_antiglitch_timeout_us);
 Timer motorParkTimer(motor_park_timeout_us);
 
 // simulator related
+Simulator simulator(PotOption::speedo);
 bool simulating_last = false;
-Timer simTimer;
+Timer simTimer; // NOTE: unused
```

Comment:
Do we need it if unused?

Reply:
Dunno, it's something Soren had, so i didn't wanna just pull it in case there's some plan for it. But I marked it so if/when we do a code-cleanup sweep we can address it.